azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+trigger:
```

Comment:
Is there no way to trigger tests on PRs?

Or are we going to continue using Travis for that?

Reply:
This triggers for master and all PRs targeting it. I'll see if we should enable dev too.